src/ch04-02-supervision.md

```diff
@@ -22,8 +22,8 @@ What's the difference between default behavior and `with_force(true)` one? By de
 The reconfiguration process consists of several stages:
 ![](assets/reconfiguration.drawio.svg)
 
-1. Config validation: the configurer sends the `ValidateConfig` request to all groups and waits for responses.
-2. If all groups respond `Ok(_)` or ignore the request, the config is considered valid, and the configurer sends `UpdateConfig` to all groups. If at least one actor responds `Err(_)`, reconfiguration is aborted, and a new config isn't used.
+1. Config validation: the configurer sends the `ValidateConfig` request to all groups and waits for responses. If all groups respond `Ok(_)` or discard the request (which is the default behaviour), the config is considered valid and the configurer proceeds to the next step.
+2. Config update: the configurer sends `UpdateConfig` to all groups. If at least one actor responds `Err(_)`, reconfiguration is aborted, and a new config isn't used.
```

Comment:
> and a new config isn't used

If I am understanding the code correctly, the new config will not be used only for the actor that returned `Err(_)`. Actors that returned `Ok(())` will use the new config. Is that the case? If yes, I think we should add clarification here.

Reply:
Nope, if any actor considers the config invalid, the next stage (`UpdateConfig`) is started at all. This provides consistency between configs if needed.

Reply:
Ok, it's all about the second step. I think `UpdateConfig` can become just a regular message and by default `system.configurers` shouldn't wait until all actors have finished the update.